### ISSUE TITLE
[BUG] still some theme integration problems

### ISSUE THREAD

**Issue by mnkhprre** [tefkah/zotero-night#82]
**Describe the bug**
sometimes work sometimes not.
**To Reproduce**

**Expected behavior**

**Screenshots**

**OS (MacOS XX.xx [specify Intel or Silicon], Windows, Linux
       macOS ventura 13.0
 [specify distro/package type]:**
<img width="1440" alt="Screenshot 2022-11-05 at 01 29 17" src="https://user-images.githubusercontent.com/54448793/200084393-0f285e20-ff45-49a7-9f9e-d1d557687cbc.png">
<img width="364" alt="Screenshot 2022-11-05 at 01 30 16" src="https://user-images.githubusercontent.com/54448793/200084412-46369937-af8a-4574-ab15-93a06195e68d.png">
<img width="1440" alt="Screenshot 2022-11-05 at 01 35 42" src="https://user-images.githubusercontent.com/54448793/200084660-67663fca-a9b8-4f9a-a660-07e8572f43f5.png">

**Zotero Night version (please check that you are on the latest version)**
i am in latest verison 
**Additional context**

**Comment by tefkah** [comment]
yeah i'm sorry, the way zotero night finds new tabs is reaaaally buggy. best advice i have if you experience these problems is to reopen the tab, that should almost always work. sorry! this might work better in newer versions as I need to rewrite the extension for Zotero 7

**Comment by mnkhprre** [comment]
> yeah i'm sorry, the way zotero night finds new tabs is reaaaally buggy. best advice i have if you experience these problems is to reopen the tab, that should almost always work. sorry! this might work better in newer versions as I need to rewrite the extension for Zotero 7

do you have any opinion when will it be released?

**Comment by tefkah** [comment]
I don't know when Zotero 7 will be released! I'll let you know when i update the extension (the update would still take effect for Zotero 6)
